<ui version="4.0" >
 <class>MsgJournalBase</class>
 <widget class="QWidget" name="MsgJournalBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>81</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>1</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="TextLabel1" >
       <property name="text" >
        <string>Subject:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtSubj" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="TextLabel2" >
       <property name="text" >
        <string>Security:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSecurity" >
       <item>
        <property name="text" >
         <string>Public</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Friends-Only</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Private</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TextLabel3" >
       <property name="text" >
        <string>Mood:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbMood" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text" >
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbComment" >
       <item>
        <property name="text" >
         <string>Enable comments</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>No email comments</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Disable comments</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
